packages/widgets/src/routes/+page.svelte

```diff
@@ -315,7 +332,7 @@
 		},
 		{
 			id: "facebook/blenderbot-400M-distill",
-			pipeline_tag: "conversational",
+			pipeline_tag: "text-generation",
```

Comment:
```suggestion
			pipeline_tag: "text2text-generation",
```

apparently this is actually text2text-generation: https://huggingface.co/facebook/blenderbot-400M-distill

(yes we've wanted to merge them for a long time but i think it's difficult)

Reply:
Good catch - thanks!